jib-core/src/integration-test/java/com/google/cloud/tools/jib/registry/BlobPusherIntegrationTest.java

```diff
@@ -31,14 +31,12 @@
 /** Integration tests for {@link BlobPusher}. */
 public class BlobPusherIntegrationTest {
 
-  @ClassRule public static LocalRegistry localRegistry = new LocalRegistry(5000);
+  @ClassRule public static final LocalRegistry localRegistry = new LocalRegistry(5000);
```

Comment:
Is this still needed?

Reply:
The test in this class pushes a blob to this registry (line 47). We could use our GCP integration test project, but I think in this case we should use an isolated registry that starts from a clean state.

Reply:
oops I see.